Dockerfile.daemon

```diff
@@ -11,12 +9,12 @@ RUN npm install
 
 COPY lerna.json tsconfig.json ./
 
-COPY packages/cli/package.json packages/cli/package-lock.json ./packages/cli/
-COPY packages/core/package.json packages/cli/package-lock.json ./packages/core/
-COPY packages/common/package.json packages/common/package-lock.json ./packages/common/
-COPY packages/http-client/package.json packages/cli/package-lock.json ./packages/http-client/
-COPY packages/doctype-tile/package.json packages/doctype-tile/package-lock.json ./packages/doctype-tile/
-COPY packages/doctype-caip10-link/package.json packages/doctype-caip10-link/package-lock.json ./packages/doctype-caip10-link/
+COPY packages/ceramic-cli/package.json packages/ceramic-cli/package-lock.json ./packages/ceramic-cli/
+COPY packages/ceramic-core/package.json packages/ceramic-cli/package-lock.json ./packages/ceramic-core/
+COPY packages/ceramic-common/package.json packages/ceramic-common/package-lock.json ./packages/ceramic-common/
+COPY packages/ceramic-http-client/package.json packages/ceramic-cli/package-lock.json ./packages/ceramic-http-client/
+COPY packages/ceramic-doctype-tile/package.json packages/ceramic-doctype-tile/package-lock.json ./packages/ceramic-doctype-tile/
+COPY packages/ceramic-doctype-account-link/package.json packages/ceramic-doctype-account-link/package-lock.json ./packages/ceramic-doctype-account-link/
```

Comment:
These changes shouldn't be here. Is this PR based on develop or master?

Reply:
I modified this one to use the package names for the master branch and created an additional dockerfile to use the package names that are on develop.

Reply:
Right, we want to make a release to master today so these package names will be removed.